sky/engine/bindings/jni/dart_jni.cc

```diff
@@ -85,6 +106,13 @@ DART_NATIVE_CALLBACK_STATIC(JniClass, FromName);
 
 FOR_EACH_BINDING(DART_NATIVE_CALLBACK)
 
+ScopedJavaGlobalRef<jobject> DartJni::class_loader_;
+ScopedJavaGlobalRef<jclass> DartJni::class_clazz_;
```

Comment:
This will create a static initializer.

Can we put these values into a class that we allocate lazily?